packages/dtslint/README.md

```diff
@@ -3,9 +3,14 @@ It will install `typescript` and `eslint` for you, so this is the only tool you
 
 Lint rules new to dtslint are documented in the [docs](docs) directory.
 
-# Just looking for ExpectType and ExpectError?
+# Just looking for a type testing tool?
+
+If you are just looking for a TypeScript type testing tool, use:
+
+- [`tsd`](https://github.com/tsdjs/tsd),
+- [`eslint-plugin-expect-type`](https://github.com/JoshuaKGoldberg/eslint-plugin-expect-type),
+- or [`tstyche`](https://github.com/tstyche/tstyche).
```

Comment:
```suggestion
- [`tstyche`](https://github.com/tstyche/tstyche).
```

One last nit; if we extend this list, we probably don't want to keep having to move `or` around. A list implies `or` already.

Reply:
Oh, you have punctuation on all lines too. I would just drop them.